cmd/main.go

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
Should we remove this?

Reply:
Yeah I was wondering that myself. I thought there may be a reasong that the file was made if there are no controllers. But if this is just manager scaffolding for eventual controllers, we can dump it.

Reply:
Yeah, this is scaffolding for eventual controllers, we don't need it.